scripts/upgrade.sh

```diff
@@ -5,8 +5,8 @@
 # to the network to trigger the upgrade process.
 
 # Update the variables below to match current conditions and desired upgrade
-current_verison=v0.3.0
-new_version=v0.4.0
+current_verison=v0.4.0
+new_version=v0.4.1
```

Comment:
needs to be v0.5.0

Reply:
https://github.com/allora-network/allora-chain/pull/594